website/content/docs/configuration/client.mdx

```diff
@@ -380,6 +380,14 @@ see the [drivers documentation](/docs/drivers).
   S3 operation must complete before it is canceled. Set to `0` to not enforce a
   limit.
 
+- `decompression_size_limit` `(string: nil)` - Specifies the maximum amount of
```

Comment:
Because the type of this config value is a string, it might help users to tell them what the expected format is with an example (ex. `100GiB`). We probably have some fairly standardized language for how to describe that elsewhere?